feat(package update and remove manual: true): gitlab-runner-17.8

debasishbsws opened the pull request:
removing the manual:true now so that automation can attempt to update some time which will result in some failures but most should pass. Instead of currently with manual:true where all updates need to be done manually

Package update to 17.8.3

debasishbsws committed:
feat(package update and remove manual): removing the manual:true now so that automation can attempt to update some time which will result in some failures but most should pass. Instead of currently with manual:true where all updates need to be done manually

Package update to 17.8.3

Signed-off-by: Debasish Biswas <debasishbsws.dev@gmail.com>

debasishbsws committed:
Fix: update base image vars tag and commit SHA

Signed-off-by: Debasish Biswas <debasishbsws.dev@gmail.com>

OddBloke commented on gitlab-runner-17.8.yaml:
```diff
@@ -135,8 +135,6 @@ subpackages:
 
 update:
   enabled: true
-  # Requires manual steps when updating
-  manual: true
```
I think these lines should be removed in your follow-up PR: until that lands, I think we will still need to manually verify any new versions that come in.

debasishbsws replied:
I have added the check for this, I will also include this check for all the other package when I will add the entrypoint subpackages

debasishbsws replied:
I think we can remove the manual:true with this check

debasishbsws committed:
feat: Add verification for base-images-tag to ensure consistency with upstream
- Added a pipeline step to verify that BASE_IMAGES_TAG matches the expected upstream value.
- The build does not fail automatically if the extracted tag is incorrect, so a manual check is necessary.
- This ensures consistency with the upstream configuration and prevents accidental mismatches.
- If the expected and extracted tags do not match, the build will fail with an explicit error message.

Signed-off-by: Debasish Biswas <debasishbsws.dev@gmail.com>
